projects/com.oracle.truffle.llvm.nodes/src/com/oracle/truffle/llvm/nodes/memory/load/LLVMDirectLoadNode.java

```diff
@@ -43,11 +43,11 @@
 import com.oracle.truffle.llvm.types.memory.LLVMHeap;
 import com.oracle.truffle.llvm.types.memory.LLVMMemory;
 
-public abstract class LLVMLoadDirectNode {
+public abstract class LLVMDirectLoadNode {
```

Comment:
These kind of stylistic changes are fine, but try to separate them from semantic changes in commits.